### ISSUE TITLE
Unable to configure Astrometry.net

### ISSUE THREAD

**Issue by His-Holy-Noodliness** [mworion/MountWizzard4#230]
I had some issues solving with ASTAP last night, so I downloaded full ANSVR indices and set up a local astrometry.net, it's working fine when used in other clients, but in MW4 I can't get the solver button to go green so I can build a model. The ansvr is running under Cygwin on Windows 10.

I've got indices in (and MW4 pointing to) C:\Users\Paul\AppData\Local\cygwin_ansvr\usr\share\astrometry\data then for the first MW4 config item (program folder) I've got C:\Program Files (x86)\Astrometry.net Local Solver but I have also tried putting the bin folder from the cygwin install there (C:\Users\Paul\AppData\Local\cygwin_ansvr\bin) and whichever I specify the solver never turns green in MW4 and I can't model.

How do I get this working please?

**Comment by mworion** [comment]
Hi,
Sorry to hear that you got problems with ASTAP. Did it work in the past? ASTAP normally does not fail. In case your changed focal length etc. please make sure that your settings in MW4 and / or your imaging application are correct. Otherwise ASTAP as well as other plate solvers could have problems. You also could increase the search radius in MW4 for ASTAP under driver setting up to 180 for doing an all sky search.
I have to say sorry about ansvr as it implements windows specific interfaces. MW4 does not support them. I kept the option open for users who want to try it with a complete linux subsystem on windows.
So still the best way is to get ASTAP again working. If you need more help, please let me know. 
If you have a log file from a failing attempt, please post it. I probable could tell you what’s going wrong in the setup.
Michel

**Comment by His-Holy-Noodliness** [comment]
Hi thanks for the reply. The weather hasn't been the good the past few days, but I'll post a log if it happens again, I tried in a linux subsystem for windows but although I built Astrometry.net after some dependency issues, MW4 still wouldn't seem to talk to it, so I switched back to ASTAP (which has, mostly been working OK for me).

BTW when I tried in a linux subsystem, should I have also installed MW4 there? I was running MW4 natively in windows, and on the same machine running astrometry.net in the linux subsystem.
